Apps/Scigraph/dwim/extensions.lisp

```diff
@@ -99,221 +64,31 @@ advised of the possiblity of such damages.
 (defun getenv (string)
   "Get the value of the environment variable named STRING."
```

Comment:
all getenv stuff may be replaced with uiop:getenv or osicat:environment-variable (the latter requires ffi, but is fully portable, while uiop has feature-conditions for each supported implementation).